prisma/migrations/20240126235456_v0_7_0/migration.sql

```diff
@@ -33,6 +37,10 @@ ADD COLUMN     "accessToken" BYTEA NOT NULL,
 DROP COLUMN "refreshToken",
 ADD COLUMN     "refreshToken" BYTEA;
 
+-- AlterTable
+ALTER TABLE "WorkflowVersion" ADD COLUMN     "checksum" TEXT NOT NULL,
```

Comment:
why dont we just default to a random string so we can migrate existing volumes and it will just update on next seen version?

Reply:
Done.